applications/MeshingApplication/custom_processes/metrics_spr_error_process.cpp

```diff
@@ -0,0 +1,591 @@
+// KRATOS  __  __ _____ ____  _   _ ___ _   _  ____ 
+//        |  \/  | ____/ ___|| | | |_ _| \ | |/ ___|
+//        | |\/| |  _| \___ \| |_| || ||  \| | |  _ 
+//        | |  | | |___ ___) |  _  || || |\  | |_| |
+//        |_|  |_|_____|____/|_| |_|___|_| \_|\____| APPLICATION
+//
+//  License:		 BSD License
+//                       license: MeshingApplication/license.txt
+//
+//  Main authors:    Anna Rehr
+//  Co-author   :    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_processes/metrics_spr_error_process.h"
+#include "utilities/variable_utils.h"
+#include "linear_solvers/skyline_lu_factorization_solver.h"
+
+namespace Kratos
```

Comment:
what does this do? we would need a short description. Also why do you need to inlcude the linear solver?

Reply:
Because we need to solve a system 9x9 for the contact SPR

I will add detailed description when everything works (not contact right now)